### PR TITLE
Update BlueRSA to community version

### PR DESCRIPTION
**Pull request by dannys42**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -28,7 +28,7 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(url: "https://github.com/Kitura/BlueRSA.git", from: "1.0.31"),
+        .package(url: "https://github.com/Kitura/BlueRSA.git", from: "1.0.200"),
         .package(url: "https://github.com/Kitura/BlueCryptor.git", from: "1.0.0"),
```
Comment (mbarnach) [inline Package.swift:32]: https://github.com/Kitura/CommonCrypto.git needs to be updated (new tag) before merging this PR, as it seems the package manager for Swift 4 is confused by the old origin (IBM-Swift).
Then the tag probably need to be propagated to BlueCryptor and updated here also.

File: Package.swift [hunk 2]
```diff
@@ -28,7 +28,7 @@ let package = Package(
...
         .package(url: "https://github.com/Kitura/BlueECC.git", from: "1.1.0"),
         .package(url: "https://github.com/Kitura/LoggerAPI.git", from: "1.7.0"),
```
Comment (mbarnach) [inline Package.swift:34]: The logger API also needs to be updated before merging this PR, with a new tag.

**Review by mbarnach — COMMENTED** [review]
We need to update the dependencies first, as it seems Travis is failing with the Swift version 4.0, due to multiple packages with different origin (IBM-Swift and Kitura).

File: Package.swift (hunk 2 above)
Comment (dannys42) [reply Package.swift:34]: Thanks @mbarnach I'm aware.  I'm just through each dependency and re-triggering CI here to confirm.  Sorry for the noise.

**Pull request merged**